src/components/Settings.vue

```diff
@@ -47,7 +47,7 @@
         </div>
         <div class="value">
           <div class="value-container">{{ transactionDeadline }} {{ t('dexSettings.min') }}</div>
-          <s-slider class="value-slider" :value="transactionDeadline" :showTooltip="false" @change="handleSetTransactionDeadline" />
+          <s-slider class="value-slider" :value="transactionDeadline" :showTooltip="false" @input="handleSetTransactionDeadline" />
```

Comment:
Why did you change commented template here?

Reply:
Sorry, I'm about to revert it.